restli-server/src/main/java/com/linkedin/restli/internal/server/ServerResourceContext.java

```diff
@@ -46,6 +46,17 @@
  */
 public interface ServerResourceContext extends ResourceContext
 {
+  /**
+   * Local attribute key for request cookies. Value should be a {@link List} of {@link HttpCookie} objects.
+   */
+  public static final String CONTEXT_COOKIES_KEY = "$Cookies$";
```

Comment:
It looks like there are two existing patterns for these attribute names:
R2 attributes are "ALL_CAPS"
Only attribute in rest.li uses the convention "classname.attributename"

As the attribute names are shared across the framework and also the application code, I like the second approach.

So these params would be:
`ServerResourceContext.class.getCannonicalName() + ".cookie".`

Reply:
Fixed